### ISSUE TITLE
error: : Package 'Statiq.App' is incompatible with 'all' frameworks in project 

### ISSUE THREAD

**Issue by gortok** [statiqdev/Statiq.Framework#118]
I went through the quickstart guide located at https://static.dev/framework

I received the following error on Step 3: Install Statiq.App:

```
 gjstocker@Georges-Air  ~/src/lwidn-newsletter/LwidnGenerator   stable  dotnet add package Statiq.App
  Determining projects to restore...
  Writing /var/folders/z3/yy8k4n8s35j7vkl3wsgkbfj80000gn/T/tmpNj2g7X.tmp
info : Adding PackageReference for package 'Statiq.App' into project '/Users/gjstocker/src/lwidn-newsletter/LwidnGenerator/LwidnGenerator.csproj'.
info : Restoring packages for /Users/gjstocker/src/lwidn-newsletter/LwidnGenerator/LwidnGenerator.csproj...
info :   GET https://api.nuget.org/v3-flatcontainer/statiq.app/index.json
info :   OK https://api.nuget.org/v3-flatcontainer/statiq.app/index.json 105ms
error: Unable to find a stable package Statiq.App with version (>= 0.0.0)
error:   - Found 46 version(s) in nuget.org [ Nearest version: 1.0.0-beta.17 ]
error: Package 'Statiq.App' is incompatible with 'all' frameworks in project '/Users/gjstocker/src/lwidn-newsletter/LwidnGenerator/LwidnGenerator.csproj'.
```

The specific error message is `Package Static.App is incompatible with 'all' frameworks in project`.

how do I fix this error message?

The important thing is that if I specify a pre-release version, it does fine. If I drop the `--version x.y.z`; it fails.

> The quickstart guide says "Use whatever is the most recent version of Statiq.App. The --version flag is needed while the package is pre-release.

I think I had assumed that there'd be at least one initial version that isn't pre-release. It looks like that is not the case? And therefore --version flag is *always* needed?
Here is output of `dotnet --info`:

```
 gjstocker@Georges-Air  ~/src/lwidn-newsletter/LwidnGenerator   stable  dotnet --info
.NET Core SDK (reflecting any global.json):
 Version:   3.1.301
 Commit:    7feb845744

Runtime Environment:
 OS Name:     Mac OS X
 OS Version:  10.15
 OS Platform: Darwin
 RID:         osx.10.15-x64
 Base Path:   /usr/local/share/dotnet/sdk/3.1.301/

Host (useful for support):
  Version: 3.1.5
  Commit:  65cd789777

.NET Core SDKs installed:
  3.0.100 [/usr/local/share/dotnet/sdk]
  3.0.101 [/usr/local/share/dotnet/sdk]
  3.1.100 [/usr/local/share/dotnet/sdk]
  3.1.301 [/usr/local/share/dotnet/sdk]

.NET Core runtimes installed:
  Microsoft.AspNetCore.App 3.0.0 [/usr/local/share/dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 3.0.1 [/usr/local/share/dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 3.1.0 [/usr/local/share/dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 3.1.5 [/usr/local/share/dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 2.1.13 [/usr/local/share/dotnet/shared/Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.1.14 [/usr/local/share/dotnet/shared/Microsoft.NETCore.App]
  Microsoft.NETCore.App 3.0.0 [/usr/local/share/dotnet/shared/Microsoft.NETCore.App]
  Microsoft.NETCore.App 3.0.1 [/usr/local/share/dotnet/shared/Microsoft.NETCore.App]
  Microsoft.NETCore.App 3.1.0 [/usr/local/share/dotnet/shared/Microsoft.NETCore.App]
  Microsoft.NETCore.App 3.1.5 [/usr/local/share/dotnet/shared/Microsoft.NETCore.App]
```

**Comment by daveaglick** [comment]
> I think I had assumed that there'd be at least one initial version that isn't pre-release. It looks like that is not the case? And therefore --version flag is always needed?

Yep, that's exactly it. None of Statiq is actually version 1.0 yet (Framework is getting really close on it's own, but I don't want to lock it in and restrict breaking API changes until I've finished up some pending downstream work in Statiq Web and Statiq Docs just in case they require some upstream API changes). Unfortunately it's a NuGet-ism that prerelease packages require a version flag to install. At one point I had the docs with the actual version (instead of x.y.z) but I kept forgetting to keep the docs up to date with the latest version.
